{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2022-2953",
      "description": "LibTIFF 4.4.0 has an out-of-bounds read in extractImageSection in tools/tiffcrop.c:6905, allowing attackers to cause a denial-of-service via a crafted tiff file. For users that compile libtiff from sources, the fix is available with commit 48d6ece8.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2022-2953",
      "reportURL": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-2953.json",
      "timeStamp": "2022-09-02T19:36:00Z"
    },
    {
      "id": "CVE-2022-2953",
      "description": "LibTIFF 4.4.0 has an out-of-bounds read in extractImageSection in tools/tiffcrop.c:6905, allowing attackers to cause a denial-of-service via a crafted tiff file. For users that compile libtiff from sources, the fix is available with commit 48d6ece8.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2022-2953",
      "reportURL": "https://gitlab.com/libtiff/libtiff/-/issues/414",
      "timeStamp": "2022-09-02T19:36:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2022-2953",
      "url": "https://gitlab.com/libtiff/libtiff/-/issues/414",
      "added_at": "2022-08-30T05:31:48Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2022-2953",
      "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-2953.json",
      "added_at": "2022-09-02T19:36:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2022-2953",
      "url": "https://gitlab.com/libtiff/libtiff/-/issues/414",
      "added_at": "2022-08-30T05:31:48Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
